CODEOWNERS

```diff
@@ -0,0 +1,169 @@
+/src/lib/merkle_ledger/ @cmr
+/src/lib/merkle_list/ @johnwu93 @cmr
+/src/lib/merkle_mask/ @nholland94 @psteckler
+/src/lib/module_version/ @psteckler @mrmr1993
+/src/lib/network_peer/ @nholland94 @cmr
+/src/lib/network_pool/ @enolan
+/src/lib/non_empty_list/ @bkase
+/src/lib/non_zero_curve_point/ @rbkhmrcr @vanishreerao
+/src/lib/o1trace/ @cmr
+/src/lib/otp_lib/ @nholland94
+/src/lib/outside_pedersen_image/ @rbkhmrcr @vanishreerao
+/src/lib/parallel/ @bkase
+/src/lib/parallel_scan/ @deepthiskumar @bkase
+/src/lib/pedersen_lib/ @cmr
+/src/lib/peer_trust/ @enolan
+/src/lib/perf_histograms/ @bkase
+/src/lib/pipe_lib/ @nholland94
+/src/lib/pokolog/ @bkase @vanishreerao @rbkhmrcr
+/src/lib/ppx_coda/ @psteckler @mrmr1993
+/src/lib/ppx_snarky/ @mrmr1993
+/src/lib/ppx_util/ @mrmr1993
+/src/lib/precomputed_values/ @cmr @bkase
+/src/lib/proof_carrying_data/ @psteckler @bkase
+/src/lib/proposer/ @nholland94 @cmr
+/src/lib/protocol_state_validator/ @nholland94
+/src/lib/prover/ @nholland94 @bkase
+/src/lib/quickcheck_lib/ @nholland94 @ghost-not-in-the-shell
+/src/lib/random_oracle/ @cmr
+/src/lib/rc_pool/ @nholland94 @cmr
+/src/lib/receipt_chain_database_lib/ @johnwu93
+/src/lib/rocksdb/ @psteckler @nholland94
+/src/lib/rocksdb_database/ @psteckler @nholland94
+/src/lib/root_history/ @cmr
+/src/lib/root_prover/ @nholland94 @johnwu93
+/src/lib/rose_tree/ @bkase @nholland94
+/src/lib/secrets/ @cmr
+/src/lib/sgn/ @deepthiskumar @bkase
+/src/lib/sgn_type/ @deepthiskumar @bkase
+/src/lib/sha256_lib/ @rbkhmrcr @vanishreerao
+/src/lib/signature_lib/ @rbkhmrcr @vanishreerao
+/src/lib/snark_bits/ @bkase
+/src/lib/snark_pool/ @enolan
+/src/lib/snark_work_lib/ @deepthiskumar
+/src/lib/snark_worker/ @cmr @nholland94
+/src/lib/snark_worker_lib/ @cmr @nholland94
+/src/lib/snarkette/ @bkase
+# /src/lib/snarky (submodule!)
+/src/lib/snarky_blake2/ @rbkhmrcr @vanishreerao
+/src/lib/snarky_curves/ @rbkhmrcr @vanishreerao
+/src/lib/snarky_field_extensions/ @bkase @rbkhmrcr @vanishreerao
+/src/lib/snarky_group_map/ @rbkhmrcr @vanishreerao
+/src/lib/snarky_log/ @rbkhmrcr @vanishreerao
+/src/lib/snarky_pairing/ @rbkhmrcr @vanishreerao
+/src/lib/snarky_taylor/ @bkase @cmr @mrmr1993
+/src/lib/snarky_verifier/ @bkase @mrmr1993
+/src/lib/sparse_ledger_lib/ @bkase
+/src/lib/staged_ledger/ @deepthiskumar @cmr
+/src/lib/staged_ledger_diff/ @deepthiskumar @cmr
+/src/lib/storage/ @bkase
+/src/lib/sync_handler/ @cmr
+/src/lib/sync_status/ @johnwu93
+/src/lib/syncable_ledger/ @cmr
+/src/lib/test_mocks/ @nholland94
+/src/lib/test_util/ @nholland94
+/src/lib/time_simulator/ @bkase
+/src/lib/transaction_database/ @johnwu93 @bkase
+/src/lib/transaction_pool/ @enolan
+/src/lib/transaction_snark/ @deepthiskumar @bkase
+/src/lib/transaction_snark_scan_state/ @deepthiskumar
+/src/lib/transaction_snark_work/ @deepthiskumar
+/src/lib/transition_frontier/ @nholland94
+/src/lib/transition_frontier_controller/ @nholland94
+/src/lib/transition_frontier_controller_tests/ @nholland94
```

Comment:
Want to be code owner for this. Made most of the tests for transition_frontier_controller. I know what stubs are supposed to do what.